### ISSUE TITLE
git fetch failed with "couldn't find remote ref" when version was "origin/master" instead of master

### ISSUE THREAD

**Issue by jonnor** [readthedocs/readthedocs.org#10787]
## Details:

* Project URL: https://readthedocs.org/projects/emlearn/
* Build URL(if applicable): https://readthedocs.org/projects/emlearn/builds/22093123/
* Read the Docs username(if applicable): jonnor

## Expected Result

Build complete successfully, as it has for several months/years

## Actual Result

Fails in git checkout step.

The latest build fails with

```
Error

There was a problem with Read the Docs while building your documentation. Please try again later. If this problem persists, report this error to us with your build id (22093123).
```

Previous builds for around 1 month fails with

```
git fetch origin --force --prune --prune-tags --depth 50 refs/heads/origin/master:refs/remotes/origin/origin/master
fatal: couldn't find remote ref refs/heads/origin/master
```
I tried running this command in my local checkout, and it also fails.

Before that, the builds succeeded. There have been no changes to the git repository structure, readthedocs configurations between it working and failing. So I am not sure what caused this, or what I can do to fix it?

**Comment by jonnor** [comment]
I noticed on my "versions" tab that it said "origin/master" for "latest". That "origin/" part seems extraneous, and in the refs there is a double origin/origin. I tried fiddling around now to remove that - and now the builds seems to pass the stage it failed at. Example: https://readthedocs.org/projects/emlearn/builds/22093413/

I do not know how that origin/ part ended up there - or if this could potentially affect other builds/projects also.
